src/Symfony/Component/Mailer/Event/QueuingMessageEvent.php

```diff
@@ -20,7 +20,7 @@
  *
  * @author Fabien Potencier <fabien@symfony.com>
  */
-final class QueuingMessageEvent extends MessageEvent
+final class MessageEvent extends MessageEvent
```

Comment:
I bet you meant to remove this file instead, right?

Reply:
good game :)